source/Mlos.Proto/OptimizerMonitoringService.proto

```diff
@@ -195,3 +206,143 @@ message Empty
 {
 };
 
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//                                                                                                                      //
+//                                                  SimpleHypergrids                                                          //
+//                                                                                                                      //
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+// The code below defines all types required to serialize a SimpleHypergrid. For now these types are not used by any of the 

```

Comment:
outdated comment

Reply:
[zaporter/grpc 874bb44d] Updated comments in OptimizerService and OptimizerMonitoringService

Reply:
[zaporter/grpc 874bb44d] Updated comments in OptimizerService and OptimizerMonitoringService